content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: परीक्षण उपयोगिताएँ
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**आयात करना**
 
 ```javascript
 import ReactTestUtils from 'react-dom/test-utils'; // ES6
-var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
+var ReactTestUtils = require('react-dom/test-utils'); // ES5, npm के द्वारा
 ```
 
-## Overview {#overview}
+## अवलोकन {#overview}
 
-`ReactTestUtils` makes it easy to test React components in the testing framework of your choice. At Facebook we use [Jest](https://facebook.github.io/jest/) for painless JavaScript testing. Learn how to get started with Jest through the Jest website's [React Tutorial](https://jestjs.io/docs/tutorial-react).
+`ReactTestUtils` आपकी पसंद के परीक्षण ढांचे में React कौम्पोनॅन्ट का परीक्षण करना आसान बनाता है। Facebook में हम व्यथाहीन जावास्क्रिप्ट परीक्षण के लिए [Jest](https://facebook.github.io/jest/) का उपयोग करते हैं। Jest वेबसाइट के माध्यम से Jest के साथ शुरुआत करने का तरीका जानें [React ट्यूटोरियल](https://jestjs.io/docs/tutorial-react).
```

Comment:
`व्यथाहीन` => `पीड़ारहित`
How about this ?

Reply:
Agree. Done.